README.md

```diff
@@ -35,60 +35,22 @@ Our ultimate goal is to build a centralized platform to host, participate and co
 
 Some background: Last year, the [Visual Question Answering Challenge (VQA) 2016](http://www.visualqa.org/vqa_v1_challenge.html) was hosted on some other platform, and on average evaluation would take **~10 minutes**. EvalAI hosted this year's [VQA Challenge 2017](https://evalai.cloudcv.org/featured-challenges/1/overview). This year, the dataset for the [VQA Challenge 2017](http://www.visualqa.org/challenge.html) is twice as large. Despite this, we’ve found that our parallelized backend only takes **~130 seconds** to evaluate on the whole test set VQA 2.0 dataset.
 
-## Installation Instructions
+## Development setup
 
-Setting up EvalAI-ngx on your local machine is really easy.
-Follow this guide to setup your development machine.
+Use [Docker Compose](https://docs.docker.com/compose/install/) to run all the components of EvalAI-ngx together. The steps are:
 
-Get the source code on your machine via git
-```
-git clone git@github.com:Cloud-CV/EvalAI-ngx.git
-```
-If you have not added [ssh key](https://help.github.com/articles/adding-a-new-ssh-key-to-your-github-account/) to your GitHub account then get the source code by running the following command
-```
-git clone https://github.com/Cloud-CV/EvalAI-ngx
-```
-
-```
-npm install -g @angular/cli
-cd EvalAI-ngx/
-npm install
-```
-
-## Development
-
-### For Running on localhost:
-
-Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
-
-### Backend for localhost:
-
-Setting up EvalAI on your local machine is really easy. You can setup EvalAI using docker:
-The steps are:
-
-1. Install [docker](https://docs.docker.com/install/linux/docker-ce/ubuntu/) and [docker-compose](https://docs.docker.com/compose/install/) on your machine.
-
-2. Get the source code on to your machine via git.
+1. Get the source code on to your machine via git.
 
     ```shell
-    git clone https://github.com/Cloud-CV/EvalAI.git evalai && cd evalai
+    git clone https://github.com/Cloud-CV/EvalAI-ngx.git && cd evalai-ngx
     ```
 
-3. Build and run the Docker containers. This might take a while.
+2. Build and run the Docker containers. This might take a while. You should be able to access EvalAI at `localhost:8888`.
 
     ```
-    docker-compose up --build
+    docker-compose up
     ```
 
-4. That's it. Open web browser and hit the url [http://127.0.0.1:8888](http://127.0.0.1:8888). Three users will be created by default which are listed below -
-    
-    **SUPERUSER-** username: `admin` password: `password`  
```

Comment:
I thinkm these credentials should be present, to let developers know about it.

Reply:
Good catch, Thanks.